### ISSUE TITLE
Disable only right tooltip

### ISSUE THREAD

**Issue by desmeit** [loonix/another_xlider#24]
Is it possible to disable only the right tooltip?

Thanks

**Comment by loonix** [comment]
You might want to not choose range slide and just user the single that way you only get 1 tooltip... But tell me what you are trying to do and I might help out if required I can code that for you in case it makes sense

**Comment by desmeit** [comment]
I would like to have a slider similar to this:

![Bildschirmfoto 2023-05-04 um 16 46 12](https://user-images.githubusercontent.com/9454529/236487144-2fb2f865-b566-4db2-a645-9bd7e3f86dff.png)

I want to show the difference to a certain value. But for this I don't need a right handler and also no tooltip.
I can disable the right handler but with the tooltips I can only customize both together.

**Comment by loonix** [comment]
On the top of my head I think in `FlutterSliderTooltip` there is a disabled property

**Comment by loonix** [comment]
https://github.com/loonix/another_xlider/blob/master/lib/models/tooltip/tooltip.dart

**Comment by desmeit** [comment]
yes but it disables both: the left and the right tooltip. it would need a setting to address both individually. like the lefthandler and the righthandler.

**Comment by loonix** [comment]
ok, I wont be able to do it now, but will have a look later. If you want in the meanwhile feel free to make a PR.

**Comment by loonix** [comment]
Maybe will create an enum like:
```dart
enum DisableTooltip {
  None,
  All,
  Left,
  Right
}
```

it will be default to None.

**Comment by desmeit** [comment]
sounds good.
